### DISCUSSION TITLE
Unknown codespaces running

### DISCUSSION THREAD

**Discussion by jainam03** [community/community#112135] [Codespaces]
Hey there!
From past few months I've been receiving emails from GitHub stating that I've used some 90% percent codespace services for my account. But when I visit [(https://github.com/codespaces)] to check if I have any running codespaces or not. Upon landing on that page I cannot see any codespaces running. I'm not sure what exactly the issue is or if there is any codespace really running or not. 
Can someone please help me out with this?

I'm attaching the mail and the codespaces page screenshots for reference.

![image](https://github.com/community/community/assets/91124384/b7132d95-9b4e-4a57-989b-f3e3989b59a2)

![image](https://github.com/community/community/assets/91124384/647f4bf9-6a04-4a3f-98a5-e0261c3895e4)

**Comment by doug1043** [comment]
Delay in updating: Sometimes, there might be a delay in updating the information on the GitHub page. If you've just ended a codespace, it may take some time for the page to reflect this.

Hidden codespaces: Check if there are hidden or inactive codespaces. They might not be running, but still count towards your codespace allocation.

Automated activities: Verify if you have scripts or automations that are automatically consuming codespace in the background.

Collaboration activities: If you're collaborating on other people's projects, the codespaces used in those projects also count towards your codespace allocation.

System errors: There might be a system error or a temporary issue on GitHub's side causing this discrepancy. In this case, contacting GitHub support might be helpful.

I suggest you check all these possibilities, and if you don't find a solution, reach out directly to GitHub support. They'll be more than happy to help you resolve this issue.
